apps/backoffice-v2/src/pages/Statistics/Statistics.page.tsx

```diff
@@ -1,15 +1,23 @@
-import React, { FunctionComponent } from 'react';
+import React, { FunctionComponent, useCallback, useState } from 'react';
 import { UserStatistics } from '@/pages/Statistics/components/UserStatistics/UserStatistics';
 import { PortfolioRiskStatistics } from '@/pages/Statistics/components/PortfolioRiskStatistics/PortfolioRiskStatistics';
 import { WorkflowStatistics } from '@/pages/Statistics/components/WorkflowStatistics/WorkflowStatistics';
+import { useStatisticsQuery } from '@/pages/Statistics/statistics.query';
+import { Loader2 } from 'lucide-react';
 
 export const Statistics: FunctionComponent = () => {
+  const { data } = useStatisticsQuery();
+
+  if (!data) {
+    return <Loader2 className={'w-4 animate-spin'} />;
+  }
+
   return (
     <div>
       <h1 className={'pb-5 text-2xl font-bold'}>Statistics</h1>
       <div className={'flex flex-col space-y-8'}>
         <UserStatistics fullName={'John Doe'} />
-        <PortfolioRiskStatistics />
+        <PortfolioRiskStatistics data={data} />
```

Comment:
I think we should explicitly pass the specific properties from data that are in use. Like this its not too clear what the component needs and since props live on a props object there's an added nesting level.